DOCS/man/options.rst

```diff
@@ -6180,15 +6180,15 @@ them.
         wide/unusual gamuts get automatically adapted to BT.709, while standard
         gamut (i.e. BT.601 and BT.709) content is not touched. (default)
     bt.470m
-        ITU-R BT.470 M
+        ITU-R BT.470 System M, CIE C whitepoint
     bt.601-525
         ITU-R BT.601 (525-line SD systems, eg. NTSC), SMPTE 170M/240M
     bt.601-625
         ITU-R BT.601 (625-line SD systems, eg. PAL/SECAM), ITU-R BT.470 B/G
     bt.709
         ITU-R BT.709 (HD), IEC 61966-2-4 (sRGB), SMPTE RP177 Annex B
     bt.2020
-        ITU-R BT.2020 (UHD)
+        ITU-R BT.2020 (UHD), that is the usual HDR space
```

Comment:
This comment seems unnecessary to me - somebody using this option should know what they're doing, no?

Reply:
No. Everybody is using display-p3, or dci-p3 (though looks like in that case it was real DCI whitepoint, wow). See this for example: